tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRFileOverride.java

```diff
@@ -8,6 +8,13 @@
 
 public class CLDRFileOverride extends CLDRFile {
 
+    /**
+     * Silently ignore missing paths for the locale used for the Comparison Value column in Survey
+     * Tool. This happens for some light-speed unit paths with @case="accusative", which have values
+     * for some locales such as Amharic ("am") but not for English.
+     */
+    static final String COMPARISON_LOCALE = "en";
```

Comment:
if it's fixed, is this still needed?

Reply:
This class is currently only used in this situation, but it's named as if it were more generic (and indeed should be merged with delegatingxmlsource in cldr-apps).  

so i am wondering if this should be a TODO with a future ticket.

Reply:
I don't know the implications well enough to address the TODO question. Meanwhile it's worthwhile merging something like this PR to avoid needlessly throwing exceptions and cluttering the logs.